ui/components/app/flask/snap-avatar/snap-avatar.js

```diff
@@ -50,10 +50,10 @@ const SnapAvatar = ({ snapId, className }) => {
       position={BadgeWrapperPosition.bottomRight}
     >
       {iconUrl ? (
-        <AvatarFavicon size={Size.LG} src={iconUrl} name={friendlyName} />
+        <AvatarFavicon size={Size.MD} src={iconUrl} name={friendlyName} />
```

Comment:
Why are we changing these back? 🤔

Reply:
We discussed with @eriknson and it allows us to gain some space here for the version :)

Reply:
Ye, after some fine-tuning and the new right-end version label, spacing looked more aligned this way